### PR TITLE
Add lemonde.fr change password url and password rule

### PR DESCRIPTION
**Pull request by highfivedenis**
Add the French newspaper 🇫🇷 📰 **Le Monde** change password URL and password rule

### CODE REVIEW

File: quirks/change-password-URLs.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "tripit.com": "https://tripit.com/account/edit/section/change_password",
     "twitch.tv": "https://www.twitch.tv/settings/security",
     "ea.com": "https://myaccount.ea.com/cp-ui/security/index",
-    "fnac.com": "https://secure.fnac.com/account/update-password",
```
Comment (highfivedenis) [inline quirks/change-password-URLs.json:5]: Duplicate

**Comment by rmondello** [comment]
Hello @highfivedenis! Can you fix the merge conflict? Then I’d be happy to take a look.

**Comment by highfivedenis** [comment]
@rmondello it's done :) Have a nice day!

**Comment by DerJuulsn** [comment]
I think this one can be closed. 
The only rule is to have more than 6 chars (which is redundant) and the change password site is a reset password site.

**Comment by highfivedenis** [comment]
@rmondello My bad, it's fixed!

**Pull request merged**